mlflow/keras.py

```diff
@@ -375,7 +389,7 @@ def _save_custom_objects(path, custom_objects):
         cloudpickle.dump(custom_objects, out_f)
 
 
-def _load_model(model_path, keras_module, **kwargs):
+def _load_model(model_path, keras_module, save_format: str, **kwargs):
```

Comment:
```suggestion
def _load_model(model_path, keras_module, save_format, **kwargs):
```

Only type-annotating save_format looks a bit weird to me. Should we remove it?

Reply:
Sure!